diff -up ags_3.6.3.2_source/Engine/ac/characterinfo_engine.cpp.omv~ ags_3.6.3.2_source/Engine/ac/characterinfo_engine.cpp
--- ags_3.6.3.2_source/Engine/ac/characterinfo_engine.cpp.omv~	2025-10-24 05:09:26.027050924 +0200
+++ ags_3.6.3.2_source/Engine/ac/characterinfo_engine.cpp	2025-10-24 05:11:07.075331303 +0200
@@ -81,7 +81,7 @@ static void FixupCharacterLoopAndFrame(C
     if (view < 0 || view >= views.size())
         quitprintf("!Character %s is assigned a invalid view %d (valid range 1..%d)", chi->scrname, view, game.numviews);
     if (!switch_from_bad_loop && (chi->loop < 0 || chi->loop >= views[view].numLoops))
-        quitprintf("!Character %s is assigned a invalid loop %d in view %d (valid range 0..%d)", chi->scrname, chi->loop, view, views[view].numLoops - 1);
+        switch_from_bad_loop = true; // Making this fatal breaks The Crimson Diamond // quitprintf("!Character %s is assigned a invalid loop %d in view %d (valid range 0..%d)", chi->scrname, chi->loop, view, views[view].numLoops - 1);
 
     if (switch_from_bad_loop && (
         (chi->loop < 0 || chi->loop >= views[view].numLoops) ||
