diff -up ags-v.3.5.0.23/Engine/Makefile-defs.linux.omv~ ags-v.3.5.0.23/Engine/Makefile-defs.linux
--- ags-v.3.5.0.23/Engine/Makefile-defs.linux.omv~	2020-01-25 20:50:52.100099701 +0100
+++ ags-v.3.5.0.23/Engine/Makefile-defs.linux	2020-01-25 20:51:08.431099867 +0100
@@ -37,7 +37,7 @@ LIBS += -ldl -lpthread -lc -lm -lstdc++
 
 ifneq ($(USE_BUILT_IN_LIBSRC), 1)
 ALDUMB :=
-LIBS += -Wl,--push-state,-Bstatic -laldmb -ldumb -Wl,--pop-state
+LIBS += -laldmb -ldumb
 endif
 
 ifeq ($(ALLEGRO_MAGIC_DRV), 1)
